src/components/Main/MainCarousel.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowLeft, faArrowRight } from '@fortawesome/free-solid-svg-icons';
+import './MainCarousel.scss';
+
+const MainCarousel = () => {
+  const TOTAL_SLIDES = 3;
+  const [currentSlide, setCurrentSlide] = useState(0);
+  const slideRef = useRef(null);
+
+  const nextSlide = () => {
+    if (currentSlide >= TOTAL_SLIDES) {
+      setCurrentSlide(0);
+    } else {
+      setCurrentSlide(currentSlide + 1);
+    }
+  };
+
+  const prevSlide = () => {
+    if (currentSlide === 0) {
+      setCurrentSlide(TOTAL_SLIDES);
+    } else {
+      setCurrentSlide(currentSlide - 1);
+    }
+  };
+
+  useEffect(() => {
+    slideRef.current.style.transition = 'all 1.0s ease-in-out';
+    slideRef.current.style.transform = `translateX(-${currentSlide}00%)`;
+  }, [currentSlide]);
+
+  return (
+    <div className="slideBox">
+      <div className="slideDisplay" ref={slideRef}>
+        {MAIN_SLIDE.map(({ id, src, alt }) => {
+          return <img key={id} className="mainSlide" src={src} alt={alt} />;
+        })}
```

Comment:
```suggestion
        {MAIN_SLIDE.map(({ id, src, alt }) => <img key={id} className="mainSlide" src={src} alt={alt} />)}
```
arrow function에서 바로 return 할 경우에는 중괄호 및 return 키워드 생략 가능합니다!